### PR TITLE
update README

### PR DESCRIPTION
**Pull request by yuokamoto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,36 @@ UE4 Project which includes examples to use rclUE.
...
+```
+Beware that the update_project_files script considers that Unreal Engine and turtlebot3-UE are located in the same folder. If that's not the case, make a copy of the script with your correct Unreal Engine folder.
+
+If the script fails because of obsolete temporary files, remove them:
+```
+rm -r Binaries Intermediate Saved Plugins/rclUE/Binaries Plugins/rclUE/Intermediate Plugins/RapyutaSimulationPlugins/Binaries Plugins/RapyutaSimulationPlugins/Intermediate
+```
+Build the plugins and UE4 project:
+```
+make turtlebot3Editor
```
Comment (Tadinu) [inline README.md:26]: For sake of possible packaging, should we add `turtlebot3` to be `make`d also?

**Review by Tadinu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yuokamoto) [reply README.md:26]: it is better to write somewhere but make turtlebot3 take a bit long time to build.

Comment (tetorea) [reply README.md:26]: just did a clean install and the makefile is not present in the folder.
But the command line used by vs code, that can be called directly in the terminal, is :
`../UnrealEngine/Engine/Build/BatchFiles/Linux/Build.sh turtlebot3Editor Linux Development <path to turtlebot3-UE folder>/turtlebot3.uproject`

**Review by tetorea — APPROVED** [review]

**Review by cconti-rr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chalice-graeme) [reply README.md:26]: There is also the update_project_files.sh script which will generate the makefile, and a build_editor.sh script that will build it, and a run_editor.sh script which will run it.

Comment (tetorea) [reply README.md:26]: The update_project_files.sh does not create the makefile, in my case. And I don't know how I manage to get a makefile in this folder before...

Comment (tetorea) [reply README.md:26]: Finally I could fix the makefile generation when callng update_project_files.sh :
Added these lines 
```
	<ProjectFileGenerator>
		<Format>Make</Format>
	</ProjectFileGenerator>
```
in the configuration part of the file `UnrealEngine/Engine/Saved/UnrealBuildTool/BuildConfiguration.xml`

**Pull request closed without merging**